gateway/api_definition.go

```diff
@@ -262,6 +263,15 @@ func (a APIDefinitionLoader) MakeSpec(def *apidef.APIDefinition, logger *logrus.
 		logger = logrus.NewEntry(log)
 	}
 
+	// new expiration feature
+	if t, err := time.Parse(apidef.ExpirationTimeFormat, def.Expiration); err != nil {
+		logger.WithError(err).WithField("name", def.Name).WithField("Expiration", def.Expiration).
```

Comment:
code scanning is complain here to receive unsanitized user input, I think we an leave with it for now

Reply:
An example that I found to solve it like this:
`if (!Pattern.matches("[A-Za-z0-9_]+", username)) `

I think it makes the code complex. Is there any other simpler solution that you know?

Reply:
why does it makes complex @furkansenharputlu ?

Reply:
We are adding extra check if api name is in this format. It is unnecessary. However, if you tell me an important reason that I miss, I can try to add that.